pkg/types/types.go

```diff
@@ -166,3 +181,19 @@ func EventString(i interface{}) string {
 	}
 	return string(b)
 }
+
+type MountNsID struct {
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,template:ns"`
+}
+
+func (e *MountNsID) GetMountNSID() uint64 {
+	return e.MountNsID
+}
+
+type NetNsID struct {
+	NetNsID uint64 `json:"netnsid,omitempty" column:"netns,template:ns"`
+}
+
+func (e *NetNsID) GetNetNSID() uint64 {
+	return e.NetNsID
+}
```

Comment:
I think this is fine for this PR. However I'd like to create a common struct (or add in `CommonData`) to hold common data for all events: mountnsid, comm, pid, etc. in a future PR.

Reply:
BTW, @alban, we could add `HostNetwork` to the `NetNsID` struct for [this issue](https://github.com/inspektor-gadget/inspektor-gadget/pull/1336).